src/components/PostPageDetail/index.js

```diff
@@ -22,7 +22,7 @@ import {useFeedDataContext} from '../../hooks/useFeedDataContext';
 import usePostContextHook, {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
 import Header from '../../screens/FeedScreen/Header';
 import useFeed from '../../screens/FeedScreen/hooks/useFeed';
-import {createCommentParentV3, getCommentList} from '../../service/comment';
+import {createChildCommentV3, createCommentParentV3, getCommentList} from '../../service/comment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include the `createChildCommentV3` function. Ensure that this function is correctly implemented and exported in the `../../service/comment` module.
```diff
- import {createCommentParentV3, getCommentList} from '../../service/comment';
+ import {createChildCommentV3, createCommentParentV3, getCommentList} from '../../service/comment';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->